streaming/rrdpush.c

```diff
@@ -486,16 +535,51 @@ static int rrdpush_sender_thread_connect_to_master(RRDHOST *host, int default_po
               , host->program_name
               , host->program_version
     );
-
+    http[eol] = 0x00;
```

Comment:
Is this necessary? I think snprintfz() already does this.

Reply:
This code was appended, because Codacy was saying that we were given a buffer without a 0x00 termination. As I said yesterday in our talk, we are having some information from Codacy that sometimes does not make sense, and this is a great example of them.